### PR TITLE
Updated the name lists!

### PR DESCRIPTION
**Pull request by hobohime**
I took out some duplicate names and kept others as some of them are canon names and we keep cannon names in the normal pool until someone decides differently.

Didn't touch the names for wetlands and desert bc they're not implemented yet, there is no reason to limit names that cannot be gotten in game yet.

### CODE REVIEW

**Comment by capillarys** [comment]
psst i thought duplicates were meant to make a name more common? or do you think those names weren't meant to be common (just out of curiosity)

**Comment by lillithrosepup** [comment]
> psst i thought duplicates were meant to make a name more common? or do you think those names weren't meant to be common (just out of curiosity)

It does make it more common, but i asked ryos a bit ago and they are p sure its an accident that the lines were duped.

File: resources/dicts/names/names.json [hunk 1]
```diff
@@ -73,16 +73,17 @@
   "normal_prefixes": [
     "Adder", "Alder", "Ant", "Antler", "Apple", "Apricot", "Arc", "Arch", "Aspen", "Aster",
     "Badger", "Barley", "Basil", "Bass", "Bay", "Bayou", "Beam", "Bear", "Beaver", "Bee", "Beech", "Beetle", "Berry",
-    "Big", "Birch", "Bird", "Bite", "Bitter", "Bittern", "Blizzard", "Bloom", "Bud",
```
Comment (Ryos00) [inline resources/dicts/names/names.json:76]: I'd probably leave "Birch" in here since it's canon and used multiple times

**Review by Ryos00 — CHANGES_REQUESTED** [review]

**Review by Ryos00 — APPROVED** [review]

**Comment by capillarys** [comment]
ahhh big excite ty hobo for names-ing and ty ryos for ze merge!! i shall now try to get some drunkelskogs in

**Pull request merged**